src/sixLesson/Decoder.java

```diff
@@ -0,0 +1,43 @@
+package sixLesson;
+
+public class Decoder {
+  public static void main(String[] args) {
+
+    String str1 = "4A3a6B\\\\\\1\\2";
+
+    String value1 = decode(str1);
+    System.out.println("Output string1: "+value1);
+
+
+  }
+
+
+  static   String decode(String string) {
```

Comment:
This function cannot convert '/1' to '1'. Also there is no validation for correct string.